src/components/userProfile/components/UserProfile.js

```diff
@@ -12,19 +15,29 @@ const {
 
 class UserProfile extends React.Component {
     render() {
+        const { match } = this.props;
+        
         return(
             <Content className={BIG_CONTENT}>
                 <Layout className={BIG_LAYOUT}>
                     <Sider width={200} className={SIDER}>
                         <SideMenu />
                     </Sider>
                     <Content className={SM_CONTENT}>
-                        <ProfileContent />
+                        <Route path={'/profile'} exact strict component={ProfileContent} />
+                        <Route
+                            path={`${match.path}/items`}
+                            component={AddItem}
+                        />
```

Comment:
I see what you did here.

Reply:
`match` is added to any component that is displayed using `react router`. Its very similar to the global object `location` but the only difference is that it is unique to the component. It tells use the current path and other information associated with the components location in the browser. I am just leveraging it in order to make the app and codebase more dynamic. I could have easily hard coded the `items` pathname as such:
```
                        <Route
                            path={`/profile/items`}
                            component={AddItem}
                        />
```
However, this way if we decided to changed the path to `UserProfile` to lets say `/account` then we would not need to change anything for this piece of code to still work. The new path `/account/items` will automatically be picked up.